### DISCUSSION TITLE
Bridge not working as an add-on on HA using a pi4

### DISCUSSION THREAD

**Discussion by rugene76** [mrlt8/docker-wyze-bridge#219] [Q&A]
Not sure what's wrong but I've installed the bridge on a Pi4, latest versions of HA as an add-on. The bridge does not seem to work.
Here's my log:
🚀 STARTING DOCKER-WYZE-BRIDGE v1.0.2
🏠 Home Assistant Mode
2021/11/05 17:14:34 [WyzeBridge] 📚 Using 'user' from local cache...
2021/11/05 17:14:34 [WyzeBridge] 📚 Using 'cameras' from local cache...
🎬 STARTING ALL 9 CAMERAS
2021/11/05 17:14:34 [WyzeBridge] Starting rtsp-simple-server v0.17.7

The log file is variable as I try reboots and restarts. Once I did have a log that said the streams were up for a few seconds and then the immediately were reported as down.

I'm trying to use VLC to view the stream at rtsp://homeassistant.local:8554/backyard but I don't get a connection

What am I missing?
thanks

**Comment by mrlt8** [comment]
Does enabling the fresh_data setting help?

**Comment by rugene76** [comment]
Thanks for the help!I turned on fresh_data but now I get ‘ invalid credentials’. I do use special characters in my password which I am repeating as instructed. I’ve tried different combinations with no luck. Are quotes needed? thanks Sent from Mail for Windows From: mrlt8Sent: Friday, November 5, 2021 5:56 PMTo: mrlt8/docker-wyze-bridgeCc: rugene76; AuthorSubject: Re: [mrlt8/docker-wyze-bridge] Bridge not working as an add-on on HA using a pi4 (Discussion #219) Does enabling the fresh_data setting help?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by mrlt8** [reply]
That's strange since you were able to login at one time.

No, quotes are not needed.

As for escaping the characters `%` or `$`, those instructions were meant for running the bridge directly in docker.. The home assistant mode uses a different method to pass data into the container, so it may not apply?

**Comment by rugene76** [comment]


**Reply by rugene76** [reply]
Tried the newest version (1.03) in HA but still no video stream. I do get the jpg's but that's it.
here's my log:
🚀 STARTING DOCKER-WYZE-BRIDGE v1.0.3
🏠 Home Assistant Mode
2021/11/15 10:42:45 [WyzeBridge] 📚 Using 'user' from local cache...
2021/11/15 10:42:45 [WyzeBridge] ♻️ Refreshing camera data for thumbnails
2021/11/15 10:42:45 [WyzeBridge] 📚 Using 'auth' from local cache...
2021/11/15 10:42:45 [WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
2021/11/15 10:42:47 [WyzeBridge] 💾 Saving 'cameras' to local cache...
🎬 STARTING ALL 9 CAMERAS
2021/11/15 10:42:47 [WyzeBridge] Starting rtsp-simple-server v0.17.8
2021/11/15 10:42:47 [pool] ☁️ Pulling "pool" thumbnail
2021/11/15 10:42:47 [East yard] ☁️ Pulling "East yard" thumbnail
2021/11/15 10:42:47 [Living Room] ☁️ Pulling "Living Room" thumbnail
2021/11/15 10:42:47 [Kitchen] ☁️ Pulling "Kitchen" thumbnail
2021/11/15 10:42:47 [workshop] ☁️ Pulling "workshop" thumbnail
2021/11/15 10:42:47 [Front Door] ☁️ Pulling "Front Door" thumbnail

Any help is appreciated
thanks

**Reply by wayne47git** [reply]
I wrote up what I have tried on the [discussion list](https://github.com/mrlt8/docker-wyze-bridge/discussions/238). I suspect that this addon does not actually work with hassio yet.

**Reply by rugene76** [reply]
thanks, I've seen similar behavior.
